Lib/test/test_capi/test_tuple.py

```diff
@@ -14,6 +14,17 @@ class TupleSubclass(tuple):
 
 
 class CAPITest(unittest.TestCase):
+    def _not_tracked(self, t):
+        # There is no need for gc.collect()
+        # since we don't track these tuples at all.
+        self.assertFalse(gc.is_tracked(t), t)
+
+    def _tracked(self, t):
+        self.assertTrue(gc.is_tracked(t), t)
+        gc.collect()
+        gc.collect()
```

Comment:
What is the purpose of these GC collections? These tests are on the C API. I don't see a direct relationship.

Reply:
IMO such tests belong to `test_tuple`, not to `test_capi.test_tuple`.